build.gradle

```diff
@@ -11,7 +11,7 @@ plugins {
     id 'signing'
 }
 
-def jarVersion = "2.1.1"
+def jarVersion = "2.1.2"
```

Comment:
Please revert this. I'll manage the version. Because we publish a snapshot on merge to main, the first thing I do _after_ I do a release is update this (You can see this here: https://github.com/nats-io/jwt.java/pull/10) that way all subsequent merges to main until the release get published with the correct version number. So 2.1.1 _is_ the proper version. You can see in the readme at the top of the repo that the current version is 2.1.0